src/components/shared/AuthProvider.tsx

```diff
@@ -43,20 +46,27 @@ export function AuthStatus() {
       >
         Not you? Sign out
       </button>
-    </>
+    </Box>
   )
 }
 
 export function RequireAuth({ children }: WithRequiredChildren) {
   let auth = useAuth()
   let location = useLocation()
+  const [params] = useSearchParams()
+
+  const loginPath = params.size > 0 ? `/login?${params.toString()}` : '/login'
```

Comment:
Now supports GET parameters if we detect them, like `u=User;8` which would be how we could see stuff feature flagged for `User.find(8)` without being logged in (I intend to use this later to test Apple Login without turning it on for everyone else)